docs_input/basics/matlabpython.rst

```diff
@@ -43,7 +43,7 @@ Conversion Table
   | Tensor slice (contiguous) | ``A(1:4,2:5)``                         | ``A[0:5,1:6]``                                 | ``Slice({0,1}, {5,6});``                        | Slices 4 elements of the outer dimension starting at 0,                                                                |          |
   |                           |                                        |                                                |                                                 | and 5 elements of the inner dimension, starting at the second element.                                                 |          |
   +---------------------------+----------------------------------------+------------------------------------------------+-------------------------------------------------+------------------------------------------------------------------------------------------------------------------------+----------+
-  | Tensor slice (w/stride)   | ``A(1:2:end,2:3,8)``                   | ``A[::2,1:9:3]``                               | ``A.Slice({0,1}, {matxEnd,6}, {2,3});``         | Slices N elements of the outer dimension starting at the first element and picking every second element until the end. |          |
+  | Tensor slice (w/stride)   | ``A(1:2:end,2:3:8)``                   | ``A[::2,1:9:3]``                               | ``A.Slice({0,1}, {matxEnd,8}, {2,3});``         | Slices N elements of the outer dimension starting at the first element and picking every second element until the end. |          |
```

Comment:
I think this should be 9 on matx since it's exclusive

Reply:
Agree, resolved with another commit